eta/core/image.py

```diff
@@ -833,6 +834,22 @@ def from_image_labels_patt(cls, image_labels_patt):
             image_set_labels.add(cls._ELE_CLS.from_json(labels_path))
         return image_set_labels
 
+    @classmethod
```

Comment:
Why not just make this a method of the underlying `Set` class and call it `Set.from_glob_patt()`?  That way we don't have to re-write it for every `Set` subclass.

Reply:
Great point!